### ISSUE TITLE
Svelte component should use event dispatchers rather than exporting "onLoad".

### ISSUE THREAD

**Issue by Tropix126** [iconify/iconify#90]
One feature of svelte is that exporting event handlers isn't necessary. Rather than using props, you can use their native syntax with either event forwarding or dispatchers, therefore the `onLoad` property can be replaced with the following:

This is a quick example of using an event dispatcher to dispatch a custom `load` event. The second argument takes in an optional object that adds data to `event.detail`.
```html
<script>
    import { createEventDispatcher } from "svelte";

    const dispatch = createEventDispatcher();

    dispatch("load", {
        // any additional data added here will go into event.detail
    });
</script>
```
 This specific component could then be used in parent context with the native syntax:
 
 ```html
<ExampleComponent on:load={handleLoad} />
```

**Comment by cyberalien** [comment]
Thanks a lot! I'll fix that as soon as possible.

**Comment by cyberalien** [comment]
Fixed.

Also for legacy and simplicity component supports both methods (both are fired if they exist, so use one option, not both at the same time):
```jsx
<Icon icon="mdi-light:home" on:load={event => console.log(`Loaded ${event.detail.icon}`);} />
```
```jsx
<Icon icon="mdi-light:home" onLoad={icon => console.log(`Loaded ${icon}`);} />
```

**Comment by cyberalien** [comment]
Published version 2.1.0 that, among other changes, contains this fix.
